app/controllers/casework/decisions/comments_controller.rb

```diff
@@ -10,7 +10,7 @@ def form_class
       end
 
       def next_step
-        crime_application_decisions_path(@crime_application)
+        new_crime_application_send_decisions_path(@crime_application)
```

Comment:
After adding comments, caseworkers are now redirected to the send_decisions_controller which offers them an explicit choice between adding another decision or sending the current decision(s) to the provider.